.github/workflows/reusable-api-e2e.yml

```diff
@@ -52,7 +52,25 @@ jobs:
         shell: bash
         env:
           NOVU_ENTERPRISE: ${{ inputs.ee }}
-        run: cd apps/worker && pnpm start:test 2>&1 &
+        run: |
```

Comment:
An attempt to make E2E more robust when the worker hangs

Reply:
feels a bit hacky i wonder if there is a more robust way to fix it within the worker service.

Reply:
I would maybe put it to a bash script than having it in yaml directly, so the workflow is less overwhelming. 

`pnpm start:worker:test 2>&1 & ./monitor-worker.sh`